src/commands/publish/route.rs

```diff
@@ -97,7 +99,8 @@ fn create(user: &GlobalUser, project: &Project, route: &Route) -> Result<(), fai
 
     if !res.status().is_success() {
         let msg = format!(
-            "⛔ There was an error creating your route.\n Status Code: {}\n Msg: {}",
+            "{} There was an error creating your route.\n Status Code: {}\n Msg: {}",
```

Comment:
same as above; we probably want to use a new emoji::ERROR here to preserve this emoji.